backend/controllers/prisonerProfile/prisonerVisits.ts

```diff
@@ -101,6 +129,18 @@ export default ({ prisonApi, pageSize = 20 }) =>
           )
           .flat()
 
+      const statuses = cancellationReasons
+        .map((type) => ({ value: `CANC-${type.code}`, text: `Cancelled: ${type.description.toLowerCase()}` }))
+        .concat(
+          completionReasons
+            .filter((reason) => reason.code !== 'CANC' && reason.code !== 'SCH')
+            .map((type) => ({ value: type.code, text: capitalize(type.description) }))
+        )
+        .concat([
```

Comment:
then the two past and future filters where the status isn't any of the other values